package.json

```diff
@@ -93,6 +93,7 @@
 		"slick-carousel": "^1.8.1",
 		"socks-proxy-agent": "^7.0.0",
 		"string-hash": "^1.1.3",
+		"tailwind-merge": "^2.5.2",
```

Comment:
I was having trouble with the `css` and `tw` props not overwriting all the class names in the styled component for the Table cell. Decided to directly install `twMerge` and refactor this component as a simple react component with twMerge handling their overwritten class names.